### ISSUE TITLE
Mask is not active until focused

### ISSUE THREAD

**Issue by beauxjames** [candreoliveira/ngMask#18]
Mask should be immediately active on the load of the view.

If the user does not provide a constant value for ng-value then the mask does not initialize until focus is gained.

Currently, when the page loads and the phone number is masked, the string is '1231234567' and when the user adds focus to the input field, then the mask is applied '(123) 123-4567'.

Currently using Chrome edge.

**Comment by cezbatistao** [comment]
+1

i did a workaround, after i fell my model i use this to 'active' the mask:
```javascript
$timeout(function() {
   $("#birthDate").trigger('click');
}, 100, false);
```

it's works but ugle :+1:

**Comment by grantopher** [comment]
I agree, this needs to be fixed. I shouldn't have to implement a workaround.

**Comment by candreoliveira** [comment]
I think It's related with https://github.com/candreoliveira/ngMask/pull/20 and It's fixed on version 3.0.12. I'm starting to work on version 4.0.0 to compress code, fix architectural problems, improve quality and other issues.
